gradle.properties

```diff
@@ -20,7 +20,8 @@ POM_DEVELOPER_ID=mikepenz
 POM_DEVELOPER_NAME=Mike Penz
 POM_DEVELOPER_EMAIL=opensource@mikepenz.dev
 # Project-wide Gradle settings.
-org.gradle.jvmargs=-Xmx1536m
+org.gradle.jvmargs=-Xmx2048M -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options\="-Xmx2048M"
```

Comment:
can you please outline the requirement on this change?

Reply:
I get a frequent OOM error when executing `wasmJsRun` tasks, so I increase the heap memory

Reply:
thanks